core/controllers/domain_objects_validator_test.py

```diff
@@ -328,5 +337,33 @@ def test_incorrect_exp_state_domain_translations(self):
             Exception, 'The length of the image tag \'alt-with-value\' '
             'attribute value should be at least 5 characters.'
         ):
-            domain_objects_validator.validate_exploration_change(
-                incorrect_change_dict)
+            (
+                domain_objects_validator.
+                validate_exp_change_dict_and_return_exp_change_domain_obj(
+                    incorrect_change_dict)
+            )
+
+    def test_correct_exp_state_domain_content(self) -> None:
+        """Validates that the correct exp change dictionary"""
+        correct_change_dict = {
+            'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+            'property_name': exp_domain.STATE_PROPERTY_CONTENT,
+            'state_name': 'New state',
+            'old_value': state_domain.SubtitledHtml(
+                'content', '').to_dict(),
+            'new_value': state_domain.SubtitledHtml(
+                'content',
+                '<oppia-noninteractive-image filepath-with-value='
+                '"&quot;abc.png&quot;" caption-with-value="&quot;'
+                '&quot;" alt-with-value="&quot;caption&quot;">'
+                '</oppia-noninteractive-image>').to_dict()
+        }
+
+        self.assertEquals(
```

Comment:
Technically you should generate the domain object, and then check its fields one by one.

Reply:
I see, thanks for the info. I will update it.

Reply:
I have updated this part.